compiler/syntax/src/res_comments_table.ml

```diff
@@ -1486,7 +1486,47 @@ and walk_expression expr t comments =
     let after_expr, rest = partition_adjacent_trailing expr.pexp_loc after in
     attach t.trailing expr.pexp_loc after_expr;
     walk_list (cases |> List.map (fun case -> Case case)) t rest
-    (* unary expression: todo use parsetreeviewer *)
+  | Pexp_try (expr, cases, finally_expr) -> (
```

Comment:
why was this case not handled at all before?
I would expect a change that only deals with finally, here